studio/next.config.js

```diff
@@ -131,6 +131,19 @@ const nextConfig = {
             key: 'X-Frame-Options',
             value: 'DENY',
           },
+          {
+            key: 'X-Content-Type-Options',
+            value: 'no-sniff',
+          },
+          {
+            key: 'Content-Security-Policy',
+            value: `
+            frame-ancestors 'none';
+            upgrade-insecure-requests;
+            `
```

Comment:
Great work on the PR! I wanted to suggest some possible options for the policy value:

```js
{
  key: 'Content-Security-Policy',
  value: `
  frame-ancestors 'none';
  upgrade-insecure-requests;
  default-src 'self';
  script-src 'self' 'nonce-{random_string}';
  style-src 'self' 'nonce-{random_string}';
  img-src 'self' data:;
  base-uri 'self';
  form-action 'self';
  report-uri /<csp-violation-report-endpoint>;
  `
    .replace(/\s{2,}/g, ' ')
    .trim(),
}
```

Reply:
Thats too aggressive of a policy right now, we load scripts from stripe and hcaptcha which need to be configured in and we dont add nonces to inline scripts atm

Reply:
Oh, my bad, didn't know the scripts were being loaded from stripe and hcaptcha, kinda new in the repo 😅 but wanting to learn and contribute. Hope the rest of the suggestion is of value. Thank you for the feedback.